src/main/resources/init_db.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE `library_db` /*!40100 DEFAULT CHARACTER SET utf8mb3 */ /*!80016 DEFAULT ENCRYPTION='N' */;
+
+CREATE TABLE `books` (
+                         `id` int NOT NULL AUTO_INCREMENT,
```

Comment:
When creating a table in MySQL, use bigint column type for storing id.

see https://github.com/mate-academy/jv-jdbc-intro/blob/main/checklist.md